packages/core/src/styles/_type.scss

```diff
@@ -0,0 +1,97 @@
+@import "@carbon/themes/scss/themes";
+@import "@carbon/type/scss/type";
+@import "@carbon/type/scss/font-face/mono";
+@import "@carbon/type/scss/font-face/sans";
+
+@include carbon--type-reset();
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+svg.chart-svg {
+	font-family: carbon--font-family('sans');
+
+	text {
+		font-size: 12px;
+		font-weight: carbon--font-weight('regular');
+
+		&.chart-label {
+			font-family: "IBM Plex Sans Condensed", "IBM Plex Sans", sans-serif;
```

Comment:
`carbon--font-family` here too? and just confirming that the default font reset doesn't cover svg elements?

Reply:
it's overriding to use the *condensed* version of Plex for the chart labels, whereas the rest of the svg text elements should be regular ibm plex

Reply:
Is there a mixin we can use there instead of manually specifying the `font-family`?

Reply:
@cal-smith that's been addressed already
![image](https://user-images.githubusercontent.com/14989804/67119419-1ed28b00-f1b5-11e9-8df9-0277bb152c33.png)